### ISSUE TITLE
[Obsolete] MONDO:0010760 XH antigen

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#8409]
**Mondo term (ID and Label)**
MONDO:0010760 XH antigen

**Reason for deprecation**
1. OMIM merged ^ 314800
MOVED TO [260100](https://www.omim.org/entry/260100)  (which is on MONDO:0009826
2. I am not sure the remaining OMIM entry is truly a hereditary human disease.
   "PA POLYMORPHISM OF ALPHA-2-GLOBULIN"
"[Dunston and Gershowitz (1973)](https://www.omim.org/entry/260100#2) showed that Xh antigen and Pa-1 are identical, that X-linkage is ruled out by findings in 2 females, and that this is probably not a mendelian polymorphism " 

**Suggested term to consider**

none, I am wondering if these are these disease descriptions or phenotypes based on antibody positivity?

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Comment by nicolevasilevsky** [comment]
@kanems I agree, I don't think MONDO:0009826 'PA polymorphism of alpha-2-globulin' is a disease. I'll tag it for obsoletion and create a separate ticket.
